lib/ui/app_starter/app_starter.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/database/shared_preferences.dart';
+import 'package:flutter_survey/di/di.dart';
+import 'package:flutter_survey/ui/home/home_screen.dart';
+import 'package:flutter_survey/ui/login/login_screen.dart';
+
+class AppStarter extends StatelessWidget {
+  AppStarter({super.key});
+
+  final _sharedPreferences = getIt<SharedPreferencesStorage>();
```

Comment:
This access to a PreferencesStorage component from the View layer (or even the ViewModel layer) violates the Clean architecture design. Even for the simple logic, the logic should go through the ViewModel and UseCase to access Storage, Network stuff 🙏 

I know there are some works done with this approach. However, it's time to follow to correct design.